app/AppKernel.php

```diff
@@ -155,10 +154,11 @@ public function registerBundles(): array
             new Mautic\UserBundle\MauticUserBundle(),
             new Mautic\WebhookBundle\MauticWebhookBundle(),
             new Mautic\CacheBundle\MauticCacheBundle(),
+            new Mautic\MessengerBundle\MauticMessengerBundle(),
         ];
 
         $queueProtocol = $this->getParameterLoader()->getLocalParameterBag()->get('queue_protocol', '');
-        $bundles[]     = new Mautic\QueueBundle\MauticQueueBundle($queueProtocol);
+        $bundles[] = new Mautic\QueueBundle\MauticQueueBundle($queueProtocol);
```

Comment:
There are many changes that are just code style changes and are actually against Mautic's CS guide. Can these be reverted so the CI check for CS passes and reviewers has less rows to review?